riemann/tests/tx/test_tx.py

```diff
@@ -1312,6 +1370,26 @@ def test_calculate_fee(self):
 
         self.assertEqual(transaction.calculate_fee(), 33400)
 
+    def test_from_bytes(self):
+        with self.assertRaises(NotImplementedError) as context:
+            tx.DecredTx.from_bytes(self)
+
+        self.assertIn('Not Yet Implemented', str(context.exception))  # TODO
+
+    def test_witness_signing_hash(self):
+        transaction = tx.DecredTx(
+            version=self.version,
+            tx_ins=[self.tx_in],
+            tx_outs=[self.tx_out],
+            lock_time=self.lock_time,
+            expiry=self.expiry,
+            tx_witnesses=[self.witness])
+
+        test_witness_hash = transaction.witness_signing_hash()
+        expected = utils.blake256(tx.DecredTx.witness_signing(transaction))
```

Comment:
Ditto above comment. Should find a definitive answer and test that.

Reply:
Will review